test/e2e/vpc/main.tf

```diff
@@ -16,33 +16,24 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-terraform {
-  required_version = ">= 1.0.1"
-
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = ">= 3.66.0"
-    }
-    kubernetes = {
-      source  = "hashicorp/kubernetes"
-      version = ">= 2.7.1"
-    }
-    helm = {
-      source  = "hashicorp/helm"
-      version = ">= 2.4.1"
-    }
-  }
+provider "aws" {
+  region = var.region
 }
 
 terraform {
   backend "s3" {
-    bucket = "terraform-ssp-github-actions-state"
-    key    = "e2e/vpc/terraform-main.tfstate"
-    region = "us-west-2"
+    bucket = ""
+    key    = ""
+    region = ""
```

Comment:
Is this required?

Reply:
Yes, unfortunately we need those values to be empty otherwise it will throw an error

Reply:
We sure about this? This doesn't seem right....can we just use variables here with defaults?

Reply:
@askulkarni2 Correction here. You are right that we may not need this anymore. I just updated the PR now.